Ladybird/Qt/Tab.cpp

```diff
@@ -390,8 +390,9 @@ Tab::Tab(BrowserWindow* window, WebContentOptions const& web_content_options, St
     };
 
     view().on_resize_window = [this](auto const& size) {
-        m_window->resize(size.width(), size.height());
-        return Gfx::IntSize { m_window->width(), m_window->height() };
+        auto new_window_size = m_window->size() - m_view->size() + QSize { size.width(), size.height() };
```

Comment:
as I understand `m_window->size() - m_view->size()` needs to be added to the a new size to account for window border painted by Qt, right? would be good to have a comment that explains what is going on here.